### ISSUE TITLE
Request for Help: Docker hub image update breaks

### ISSUE THREAD

**Issue by shahabganji** [staff0rd/docker-crankier#1]
Hi @staff0rd, 

You've done quite an amazing work and I have to thank you for that.

I just pulled the latest docker file from the docker hub and I have observed some problems. It sounds that the one which uses `netcoreapp3.0` has exposed port 80 and I was able to connect with clients to that server both running inside a docker hub, however, the one using `netcoreapp5.0`  is running on port 5000 and it seems it does not expose that port so the clients cannot connect to the server running inside a container. Any guidelines? 

The new docker image run time: 

<img width="699" alt="Screen Shot 2020-01-04 at 08 54 36" src="https://user-images.githubusercontent.com/4405518/71760362-bb154f80-2ed1-11ea-8459-5ea3806ab8f8.png">

The old docker image run time: 

<img width="566" alt="Screen Shot 2020-01-04 at 08 56 43" src="https://user-images.githubusercontent.com/4405518/71760363-c23c5d80-2ed1-11ea-9f8b-1ea74e22b8ec.png">

I am getting an exception:

```bash
Connection.Start Failed: System.Net.Http.HttpRequestException: An error occurred while sending the request.
```

The following is the command I am using for the server: 

```
docker run -d -p 80:80 -p 5000:5000 -p 5001:5001 --name crankier staff0rd/crankier server
```

And I used the following command with ports `80`, `5000`, and `5001`, but all failed with the same above-mentioned issue:

```
docker run --rm staff0rd/crankier local --target-url http://192.168.16.63/echo --workers 10
```

```
docker run --rm staff0rd/crankier local --target-url http://192.168.16.63:5000/echo --workers 10
```

```
docker run --rm staff0rd/crankier local --target-url https://192.168.16.63:5001/echo --workers 10
```

But the old docker image works just fine.

And does this docker image only runs for `azure`, due to [these](https://github.com/staff0rd/docker-crankier/commit/7f02286d3dbc04d027b0426c8c1df015925a9857) changes?

**Comment by staff0rd** [comment]
I was able to pull `staff0rd/crankier:latest` and confim your findings.  Additionally I see that I must have been aware of the same earlier considering 8438043.  I'll merge that PR in.

**Comment by shahabganji** [comment]
@staff0rd, 
Would you mind considering two versions/tags of crankier since many people are still working with `netcoreapp3.0` and not moving to `netcoreapp5.0`  and I guess the result would be different when they want to load test their servers? It would be beneficial to have such opportunity, although I made [my own crankier](https://hub.docker.com/repository/docker/shahabganji/crankier) versions/tags for my own future tests, of course, they are just copies of what you have made earlier and totally based on them but with different tags, they point to docker images you have created, but people will definitely end up [here](https://hub.docker.com/r/staff0rd/crankier) for your docker files, so why not help them better distinguish which `.netcoreapp` is running under the hood. Thanks again for your time and effort 🙏

**Comment by staff0rd** [comment]
I was thinking the same, although the core issue is that the Azure SignalR functionality was merged into aspnet master after they shipped 3.0 and bumped the version to 5.  This means it would unavailable on the 3.0 tag.

Regardless, I’m happy if you wanna open a PR to split them.
